modules/heat_conduction/test/tests/heat_source_bar/tests

```diff
@@ -3,6 +3,7 @@
     type = 'Exodiff'
     input = 'heat_source_bar.i'
     exodiff = 'heat_source_bar_out.e'
+    abs_zero = 1e-7
```

Comment:
Yea sorry I should have noticed this in my earlier review...the exodiff value was 1.2921338e-08, so 1e-7 should do the trick now